sutraani/1/1/1.1.71.txt

```diff
@@ -2,7 +2,7 @@
     "i": "11071",
     "an": "स्वम्$11068##रूपम्$11068",
     "ad": "",
-    "ss": "अन्त्येन इता सह आदिः स्वस्य ",
+    "ss": "अन्त्येन इता सह आदिः स्वस्य रूपम्",
```

Comment:
No the original was correct, you can actually change this to रूपस्य. Vibhaktis can change when words move to other sutras.

Reply:
corrected